connectors/src/connectors/google_drive/temporal/activities.ts

```diff
@@ -468,6 +488,20 @@ export async function incrementalSync(
         authCredentials
       );
       if (driveFile.mimeType === "application/vnd.google-apps.folder") {
+        const parents = await getFileParentsForUpsert(
```

Comment:
to keep an eye on, parents calls grab from google api and may be costly
I think it'lll be fine but :eye: :)
can you update risks on the PR description?

Reply:
done, although since i'm now using the cache in getFileParents, I would expect the perfs to be better